<launch>
  <node pkg="g1_25_assign4_pkg" exec="accel_velocity_approximator" name="accel_velocity_approximator" output="screen">
    <param name="odom_frame_id" value="odom"/>
    <param name="base_frame_id" value="base_link"/>
    <param name="remove_gravity_z" value="true"/>
    <param name="use_orientation_for_gravity" value="false"/>
    <param name="gravity_m_s2" value="9.81"/>
    <param name="accel_smoothing_alpha" value="0.2"/>
    <param name="accel_deadband" value="0.02"/>
    <param name="velocity_damping" value="0.02"/>
    <param name="max_dt_seconds" value="0.2"/>
  </node>
</launch>
